ess/src/API/EMBC.ESS/Managers/Events/EvacuationFileLoader.cs

```diff
@@ -121,10 +121,6 @@ public async System.Threading.Tasks.Task Load(Shared.Contracts.Events.Support su
                     interac.SecurityQuestion = payment.SecurityAnswer;
                     interac.RelatedPaymentId = payment.Id;
                 }
-                if (support.IsSelfServe)
```

Comment:
Are we removing this Language change?

Reply:
Yes, for now. There's a tension here between the design and what is in the responders portal for e-transfer. We need to remember to revisit this a bit later